pyroclast/prototype/prototype.py

```diff
@@ -118,33 +138,63 @@ def run_minibatch(epoch, batch, phase, is_train=True):
                                   separation_coeff *
                                   tf.reduce_mean(loss_term_dict['separation']),
                                   step=global_step)
-            tf.summary.scalar(prefix + "loss/total loss",
-                              loss,
+            tf.summary.scalar(prefix + "loss/mean final loss",
+                              mean_loss,
                               step=global_step)
+        loss_numerator = tf.reduce_sum(loss)
+        accuracy_numerator = tf.reduce_sum(
+            tf.cast(tf.equal(prediction, labels), tf.int32))
+        denominator = x.shape[0]
+        return loss_numerator, accuracy_numerator, denominator
 
     ### PHASE 1
     # run training loop
-    if debug: print("PHASE 1 - TRAINING CONV STACK AND PROTOTYPES")
-    for epoch in range(epochs_phase_1):
+    print("PHASE 1 - TRAINING CONV STACK AND PROTOTYPES")
```

Comment:
We might want to consider starting to use Python loggers: https://docs.python.org/3.7/library/logging.html